pquery-run.sh

```diff
@@ -338,7 +339,7 @@ if [[ $PXC -eq 1 ]];then
   rm -rf ${BASEDIR}/my.cnf
   echo "[mysqld]" > ${BASEDIR}/my.cnf
   echo "basedir=${BASEDIR}" >> ${BASEDIR}/my.cnf
-  echo "wsrep-debug=1" >> ${BASEDIR}/my.cnf
```

Comment:
Please revert this change, wsrep-debug=ON is not supported in PXC-8.0.

Reply:
Accidently these PXC changes crept in. Reverted.